cedar-drt/fuzz/src/schemas.rs

```diff
@@ -65,7 +65,9 @@ fn action_type_equivalence(name: &str, lhs: ActionType, rhs: ActionType) -> Resu
         match (lhs.applies_to, rhs.applies_to) {
             (None, None) => Ok(()),
             (Some(lhs), Some(rhs)) => {
-                if empty_target(&rhs) || empty_target(&lhs) || rhs == lhs {
+                // If either of them has at least one empty appliesTo list, the other must have the same attribute.
+                // Otherwise both of them must apply to unspecified entities or non-empty entity lists, which must be equal.
+                if (either_empty(&lhs) && either_empty(&rhs)) || rhs == lhs {
```

Comment:
I don't see how the comment "the other must have the same attribute" is reflected in the code

Reply:
I meant the other must have the same characteristics or states.

Reply:
My interpretation was: either the two `appliesTo` specifications must be equal, or they both must satisfy `either_empty` (indicating that they're action groups)